### DISCUSSION TITLE
Genkidama - Call for contributions

### DISCUSSION THREAD

**Discussion by AbdelStark** [kkrt-labs/kakarot#83] [Q&A]
Dear Saiyajins,
We want Kakarot to be a community driven effort.
There are many ways to get involved. You can contribute to discussions around architectural choices, code, write some tests, documentation, devops, design, ...

If you are interested in joining the adventure please interact in this discussion. And tell us on which front you want to help.
- Coding
    - Kakarot Cairo code
    - Kakarot backend in Rust
    - Web application (React / Typescript)
    - Landing page / Website
- Tests
- Documentation
- Design (website, NFTs/SBT for contributors,memes,...)

**Comment by etashhh** [comment]
Hey everyone, I'm etash 👋  

I'm primarily a Solidity, Rust, and Python dev but am on the journey to learning Cairo with you all. For Kakarot, I'm interested in assisting with the Rust backend and Cairo EVM implementation 🙂

**Comment by Eikix** [comment]
Hi y'all Saiyajins, 
My name's Elias, I'm a 25-year-old fullstack engineer specialised in web3 and serverless. 
💪 My strengths are Typescript, React, AWS serverless infrastructure, Solidity and smart contract development, and recently Cairo!

👷 My interests: 
- Coding Kakarot Cairo 🔥
- I can help with React / Typescript if need be
- Happy to write some documentation
- I ❤️ memes and will retweet kakarot memes and write them 👉👈
- I love tests. I've worked extensively on serverless integration testing. Although this probably won't be useful, I just enjoy tests in general!


I can't wait to hack at it with y'all. 
Feel free to reach me on telegram @eliastzr

**Comment by 0xLucqs** [comment]
Hi all :wave:,
I'm Lucas from StarkWare one of the Kakarot core devs. 
You can add me on your PRs as a reviewer if you need it I can also help if you have cairo  :fire: or python :snake: questions.
You can reach out on twitter @lucas_lvy

**Comment by 0xMentorNotAPseudo** [comment]
Hiya 👋,
I'm Mentor from Alpha Road one of the Product Owner leading Kakarot effort.

From time to time I eat and produce code and mainly focused on the product mgmt side. 
If you want to contribute and don't know where to start please feel free to reach out on Telegram 🔥

👉 [t.me/MRMENTOR](https://t.me/MRMENTOR)

**Comment by danilowhk** [comment]
Hey guys! My name is Danilo, web3 dev and beginner in Cairo.
I am interested in contributing with:
- Kakarot Cairo code
- FrontEnd React/Typescript

**Comment by TotalPizza** [comment]
Hey,
I'm FreshPizza. :pizza: 
Solidity Dev for ~1.5 years and Cairo for ~3/4.
Would mainly like to contribute to:
- Kakarot Cairo code
- (And with that ofc update docs and write tests)

**Comment by EvolveArt** [comment]
Hey,
0xevolve,
I will try to help in my free time both on the cairo code and the web application ✨

**Comment by dcbuild3r** [comment]
Hey! I could contribute into the Rust backend, I can do some front-end changes (React, Typescript, Tailwind, etc) if needed, and write some docs possibly.

**Comment by Darlington02** [comment]
Hey guys, i'm Darlington. A 20 year old solidity dev interested in ZKPs.

I'm currently learning to write better Cairo codes, so i'd do my best to contribute to Kakarot Cairo code.

I'm also pretty good with writing technical contents, so I'm also very much interested in helping out with the documentation too.

**Comment by Kelvyne** [comment]
Hey,
I would love to help on the Cairo part of the project :)

**Comment by mattegoat** [comment]
Hey👋
Matteo, I'm a retired developer
More than happy to help with documentation, meme, design, articles

**Comment by thanhtx1178793** [comment]
Hi, I'm Thanh, solidity developer and beginner in Cairo.
I am interested in contributing with Kakarot Cairo Code.

**Comment by ClementWalter** [comment]
Hey, I'm Clément, building Starksheet

I'm primarily interested in contributing to the cairo code and doc but I can help with the FE and other things

**Comment by petarcalic99** [comment]
Hello!
I'm Petar, i have a python, AI background and i am looking to gain more dev experience.
At the moment i am most interested in progressing in Cairo and EVM thematics. So i would be most interested to work on the Cairo EVM code to begin.
In the future i would be happy to take on some rust backend, tests or docs issues.

**Comment by Riad-Quadratic** [comment]
Hello all, my name is Riad from Quadratic-labs, web3 dev. I have many years experience developing in Python and lately I'm working on a DAO in cairo. I am interested in contributing to Kakarot Cairo code.

**Comment by newgnart** [comment]
Hi all, I'm Trang, building/learning on StarkNet.
I'm interested in contributing to Cairo/Python code, and also documentation/writing if needed.

**Comment by tzamoj** [comment]
Hello Team,
Thomas here from Quadratic-labs. 
Would be interested to contribute some opcodes and tests, e.g. 0x18, 0x39, ...

@abdelhamidbakhta : could we be granted write access to submit some PRs ?

**Reply by ClementWalter** [reply]
I don't think it's necessary to have write access, fork isn't working for you?

**Comment by Flydexo** [comment]
Hello everybody,

I'm Flydexo, young French developer.

I can program both front and back.

I master Typescript, nodejs, rust, golang

I'm a beginner with contributing to open source.

**Comment by waifu007** [comment]
Hi, my name is Marlen. I'm a designer and I'm good at making stickers and memes. Perhaps I can be of use to you.
